src/main/java/build/buildfarm/instance/shard/ServerInstance.java

```diff
@@ -795,7 +795,7 @@ private Set<String> filterAndAdjustWorkersForDigest(
             .immutableCopy();
     if (!workersToBeRemoved.isEmpty()) {
       try {
-        log.log(Level.INFO, format("adjusting locations for the digest %s", digest));
+        log.log(Level.FINE, format("adjusting locations for the digest %s", digest));
```

Comment:
Separate this change if you want it

Reply:
Created new PR
https://github.com/bazelbuild/bazel-buildfarm/pull/1573